src/Json/JsonGenerator.php

```diff
@@ -171,6 +171,7 @@ protected function createDebit($transaction, $language){
             'language'             => $language,
             'referenceSchemeTransactionIdentifier' => $transaction->getReferenceSchemeTransactionIdentifier(),
             'surchargeAmount'      => $transaction->getSurchargeAmount(),
+            'senderAccountReferenceUuid' => $transaction->getSenderAccountReferenceUuid()
```

Comment:
`getSenderAccountReferenceUuid` method not found

Reply:
added missing files